client/src/components/ResultPanel/index.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { Disposable, Uri, commands, window } from "vscode";
+
+import { createWriteStream } from "fs";
```

Comment:
It would be great to use `vscode.workspace.fs` whenever possible to work across virtual file systems VS Code supports.

Reply:
Done https://github.com/sassoftware/vscode-sas-extension/pull/550/commits/7834fea3dd0fa8e2b567e73b159125587d079ba9